azerothcore/azerothcore-wotlk
# fix(Scripts/ZulGurub): Priestess Jeklik intro

## Changes Proposed:
-  Don't engage immediately, do a path first.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12081
- Closes https://github.com/chromiecraft/chromiecraft/issues/3623

## SOURCE:


## Tests Performed:

- Tested in-game.

## How to Test the Changes:


1. Go to Jeklik, and engage with her
2. She should do a little path to come down from the ledge.
3. To test the wrong form, you can .learn 57994 and cast it until you miss it (you might need a few tries). Once you miss, you should see her in the right form.

## Known Issues and TODO List:


- [x] Waypoints are not blizzlike (yet). I'm waiting for sniffs.
